setting-up-appsmith/aws-ami.md

```diff
@@ -0,0 +1,185 @@
+# Introduction
+This tutorial will walk you through the process of using the AWS Console to create and provision a new AWS cloud server. Since AWS offers a Free Tier valid for 12 months, you will be able to give Appsmith a whirl without worrying about being billed for usage.
+
+# Overview
+
+Here are the steps you’ll follow in this tutorial:
+- Register with Amazon Web Services (AWS) 
+- Generate an SSH key pair
+- Create an AWS Security Group
+- Deploy Appsmith on an AWS cloud server
+- Log in and start using Appsmith
+- Configure a custom domain for your app
+
+The next sections will walk you through these steps in detail.
+
+# Step 1: Register With Amazon Web Services (AWS)
+```
+At the end of this step, you will have signed up for the Amazon Web Services free tier. If you already have an Amazon Web Services account, you may skip this step.
```

Comment:
The following steps are AWS specific, and have no Appsmith related config / inputs. Hence, we should share link to an appropriate AWS document that guiders users to sign on this specific plan, instead of listing the steps ourselves again.
The philosophy being that 
- we need not document things that have already been documented by the owner.
- this also ensures that our documents doesn't go stale when the owner changes their product / flow.